docs/changes/units/16441.feature.rst

```diff
@@ -0,0 +1,2 @@
+Add unit foe (or Bethe, equivalent to 1e51 erg) to ``astropy.units.astrophys``.
```

Comment:
Where the unit is located is an implementation detail, since users should just get it as `u.foe`. So, could you remove " to ``astropy.units.astrophys``? Maybe,
```
Add the unit of foe (or Bethe, equivalent to 1e51 erg), often used as a unit for supernova
explosion energies.
```